mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java

```diff
@@ -0,0 +1,101 @@
+package life.qbic.projectmanagement.persistence;
+
+import life.qbic.projectmanagement.project.repository.ProjectRepository;
+import life.qbic.projectmanagement.project.Project;
+import life.qbic.projectmanagement.project.ProjectId;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+
+/**
+ * <b>Project JPA repository</b>
+ *
+ * <p>Implementation for the {@link ProjectRepository} interface.
+ *
+ * <p>This class serves as an adapter and proxies requests to an JPA implementation to interact with
+ * persistent {@link Project} data in the storage layer.
+ *
+ * <p>The actual JPA implementation is done by {@link QbicProjectRepo}, which is injected as dependency
+ * upon creation.
+ *
+ * @since 1.0.0
+ */
+@Component
+public class ProjectJpaRepository implements ProjectRepository {
+
+  private static ProjectJpaRepository INSTANCE;
```

Comment:
You can replace all the singleton stuff by addin `@Scope("singleton")` to the class